### PR TITLE
PR for providing ESLint and HoundCI screenshots

### PR DESCRIPTION
**Pull request by fmustapha**
#### What does this PR do?
Merges changes in bayo-kakashi-fmaustapha to  bayo-kakashi-fmaustapha-output. Output for checkpoint one.
#### Description of Task to be completed?
Deliver two screenshots of ESLint [Airbnb]  style violations in the terminal and [HoundCI] comment on PR.
#### How should this be manually tested?
By viewing the screenshots.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
N/A
#### Screenshots (if appropriate)
ESLint Airbnb Style Violations 
eslint-shot

HoundCI Comments
houndcishot

#### Questions:
N/A

### CODE REVIEW

File: 06-building-a-program-start-to-finish/automating-tasks/output/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#### Output
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/automating-tasks/output/README.md]: This shouldn't be here. Please remove from this PR.

File: 06-building-a-program-start-to-finish/setting-up-linters/output/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#### Output
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/setting-up-linters/output/README.md:1]: Great Job!

**Comment by q-ode** [comment]
Please fix that unnecessary file in the PR.

File: 06-building-a-program-start-to-finish/automating-tasks/output/README.md (hunk 1 above)
Comment (fmustapha) [reply 06-building-a-program-start-to-finish/automating-tasks/output/README.md]: True,  I will take it off. Thank you.

**Pull request merged**